Nexus.Sdk.Shared/Requests/CreateMailRequest.cs

```diff
@@ -11,7 +11,8 @@ public class CreateMailRequest
     public required string Type { get; set; }
 
     [JsonPropertyName("references")]
-    public MailEntityCodes? References { get; set; }
+    [Required]
+    public required MailEntityCodes? References { get; set; }
```

Comment:
Could you please remove the optional param as it is required now :) 
```suggestion
    public required MailEntityCodes References { get; set; }
```